apps/vth-frontend/src/app/[locale]/sitemap.ts

```diff
@@ -0,0 +1,68 @@
+import { MetadataRoute } from 'next';
+import { cookies } from 'next/headers';
+import { GET_ALL_ARTICLES_SLUES, GET_ALL_THEME_SLUGS, GET_NAVIGATION_PAGES } from '@/query';
+import { createStrapiURL } from '@/util/createStrapiURL';
+import { fetchData } from '@/util/fetchData';
+
+const { origin } = new URL(process.env.FRONTEND_PUBLIC_URL || 'http://localhost:3000');
+export type Attributes = {
+  slug: string;
+  updatedAt: string;
+};
+export type Urls = {
+  attributes: Attributes;
+};
+export interface GenerateUrlParameter {
+  locale?: string;
+  origin: string;
+  segment?: string;
+  urls: Urls[];
+}
+export const generateStaticPagesPath = (paths: string[], locale?: string) => {
+  const { href } = new URL(`${origin}/${locale ? `${locale}/` : ''}`);
```

Comment:
`URL` werkt met een baseURL als tweede argument.
```suggestion
  const baseURL = locale ? new URL(`/${locale}/`, origin) : origin;
```

en dan verderop:

```js
    url: new URL(path, baseURL).href,
```